src/pages/library/DataBrowser.js

```diff
@@ -55,17 +56,26 @@ const extractCatalogFilters = dataCatalog => {
         div([snapshotAccessTypes[accessKey]])
       ])]
     }, _.keys(snapshotAccessTypes)))
+  }, {
+    name: 'Data use policy',
+    labels: releasePolicyArray,
+    labelDisplays: _.zipObject(releasePolicyArray, _.map(releasePolicy => {
+      return [div({ key: `releasePolicy-filter-${releasePolicy.tag.toLowerCase()}`, style: { display: 'flex', flexDirection: 'column' } }, [
+        releasePolicy.label,
+        releasePolicy.desc && div({ style: { fontSize: '0.625rem', lineHeight: '0.625rem' } }, [releasePolicy.desc])
+      ])]
+    }, snapshotReleasePolicies))
```

Comment:
nit: How do you feel about making this a `flow`? For me it is a bit easier to read, but it may just be me.
```suggestion
    labelDisplays: _.flow(
      _.map(({ label, desc, tag }) => {
        return [div({ key: `releasePolicy-filter-${tag.toLowerCase()}`, style: { display: 'flex', flexDirection: 'column' } }, [
          label,
          desc && div({ style: { fontSize: '0.625rem', lineHeight: '0.625rem' } }, [desc])
        ])]
      }),
      _.zipObject(releasePolicyArray)
    )(snapshotReleasePolicies)
```